@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -216,13 +216,27 @@ function computePointTextOffset(
     offset.x = textElement.xOffset;
     offset.y = textElement.yOffset;
 
+    switch (placement.h) {
+        case HorizontalPlacement.Left:
+            // In practice applies: -= pointLabelMargin.x
```

Comment:
I think this comment it's difficult to understand. I would just simply mention that it's expected that any margin is already applied to the text element bounds.